CONTRIBUTING.md

```diff
@@ -0,0 +1,238 @@
+
+- Open an [Issue](https://github.com/samipak458/Simple-Library-Managment/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant.
+
+We will then take care of the issue as soon as possible.
+
+<!--
+You might want to create a separate issue tag for questions and include it in this description. People should then tag their issues accordingly.
+
+Depending on how large the project is, you may want to outsource the questioning, e.g. to Stack Overflow or Gitter. You may add additional contact and information possibilities:
+- IRC
+- Slack
+- Gitter
+- Stack Overflow tag
+- Blog
+- FAQ
+- Roadmap
+- E-Mail List
+- Forum
+-->
+
+## Basics of Git and GitHub
+
+### Git & GitHub
+
+Before we proceed, it's better to know the difference between Git and Github. Git is a version control system (VCS) that allows us to keep track of the history of our source code , whereas GitHub is a service that hosts Git projects. 
+
+We assume you have created an account on Github and installed Git on your System.
+
+Now enter your name and E-mail (used on Github) address in Git, by using following command.
+
+`$ git config --global user.name "YOUR NAME"`
+` $ git config --global user.email "YOUR EMAIL ADDRESS"`
+This is an important step to mark your commits to your name and email.
+
+<br />
+
+### Fork a project
+
+You can make a copy of the project to your account. This process is called forking a project to your Github account. On Upper right side of project page on Github, you can see -
+
+<p align="center">  <img  src="https://i.imgur.com/P0n6f97.png">  </p>
+Click on fork to create a copy of project to your account. This creates a separate copy for you to work on.
+
+<br />
+
+<br />
+
+### Clone the forked project
+
+You have forked the project you want to contribute to your github account. To get this project on your development machine we use clone command of git.
+
+`$ git clone https://github.com/samipak458/Simple-Library-Managment.git` <br/>
+Now you have the project on your local machine.
+
+<br />
+
+### Add a remote (upstream) to original project repository
+
+Remote means the remote location of project on Github. By cloning, we have a remote called origin which points to your forked repository. Now we will add a remote to the original repository from where we had forked.
+
+`$ cd <your-forked-project-folder>`
+`$ git remote add upstream https://github.com/samipak458/Simple-Library-Managment.git` <br/>
+You will see the benefits of adding remote later.
+
+<br />
+
+### Synchronizing your fork
+
+Open Source projects have a number of contributors who can push code anytime. So it is necessary to make your forked copy equal with the original repository. The remote added above called Upstream helps in this.
+
+`$ git checkout main`
+`$ git fetch upstream`
+`$ git merge upstream/main`
```

Comment:
Our main branch is named as `master` not `main`

Reply:
Not a problem bro

Reply:
Is It done?